config/main.py

```diff
@@ -3048,6 +3058,132 @@ def unbind(ctx, interface_name):
     config_db.set_entry(table_name, interface_name, None)
 
 
+#
+# 'ipv6' subgroup ('config interface ipv6 ...')
+#
+
+@interface.group()
+@click.pass_context
+def ipv6(ctx):
+    """Enable or Disable IPv6 processing on interface"""
+    pass
+
+@ipv6.group('enable')
+def enable():
+    """Enable IPv6 processing on interface"""
+    pass
+
+@ipv6.group('disable')
+def disable():
+    """Disble IPv6 processing on interface"""
+    pass
+
+#
+# 'config interface ipv6 enable use-link-local-only <interface-name>'
+#
+
+@enable.command('use-link-local-only')
+@click.pass_context
+@click.argument('interface_name', metavar='<interface_name>', required=True)
+def enable_use_link_local_only(ctx, interface_name):
+    """Enable IPv6 link local address on interface"""
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    ctx.obj = {}
+    ctx.obj['config_db'] = config_db
+    db = ctx.obj["config_db"]
+
+    if clicommon.get_interface_naming_mode() == "alias":
+        interface_name = interface_alias_to_name(db, interface_name)
+        if interface_name is None:
+            ctx.fail("'interface_name' is None!")
+
+    if interface_name.startswith("Ethernet"):
+        interface_type = "INTERFACE"
+    elif interface_name.startswith("PortChannel"):
+        interface_type = "PORTCHANNEL_INTERFACE"
+    elif interface_name.startswith("Vlan"):
+        interface_type = "VLAN_INTERFACE"
+    else:
```

Comment:
Not required for sub interface?

Reply:
As per HLD, Supporting IPv6 Link Local addresses on Ethernet, Port-channel and VLAN interfaces only.
For sub-interface support, will come with new PRs.

Reply:
Please use utility functions from [here ](https://github.com/Azure/sonic-utilities/blob/master/utilities_common/cli.py)for these checks as
`if !is_valid_vlan_interface` or ..

Reply:
Here I am checking the name of the interface only, I have used the utility function function in line 3115 to validate the interface.